src/client/mod.rs

```diff
@@ -97,6 +99,57 @@ impl GaloyClient {
         Ok(me)
     }
 
+    pub fn fetch_balance(&self, wallet: Option<Wallet>) -> anyhow::Result<String> {
+        let me = self.me()?;
+
+        let default_wallet_id = me.default_account.default_wallet_id;
+
+        let wallets = &me.default_account.wallets;
+        let mut wallet_balances = String::new();
+
+        for wallet_info in wallets {
+            match &wallet {
+                Some(Wallet::Usd) if wallet_info.wallet_currency == WalletCurrency::USD => {
+                    return Ok(format!("USD wallet: {} cents", wallet_info.balance));
+                }
+                Some(Wallet::Btc) if wallet_info.wallet_currency == WalletCurrency::BTC => {
+                    return Ok(format!("BTC wallet: {} sats", wallet_info.balance));
+                }
+                None => {
+                    if wallet_info.wallet_currency == WalletCurrency::USD {
+                        wallet_balances += &format!(
+                            "USD wallet{}: {} cents\n",
```

Comment:
are we exporting the value always as a string? would be good to have the option to expose it as a json as well (can be done in another PR later in the project)

Reply:
For the time being, the main focus of this command is to output the balance in the CLI, hence the string format. However, as you've suggested, as we further develop the cli, the necessity for the function to return a number or a JSON will definitely arise as this function will be used in other functions or tests as well.